scripts/asm-installer/install_asm

```diff
@@ -496,7 +502,11 @@ OPTIONS:
                                       omitted if not installing optional
                                       features. To add multiple files, specify
                                       them with multiple options one at a time.
-
+  --ca_name|--ca-name    <CA NAME>    Required only if --ca option is private_ca.
```

Comment:
The canonical form is just the snake_case version, don't list the hyphenated-case version as well.

Reply:
fixed